### PR TITLE
Multiple updates across files(readme, package, ncfs tests, test 212, file result view,

### PR DESCRIPTION
**Pull request by NadHodeGW**

### CODE REVIEW

File: specs/features/request-history-file-detail-view.feature [hunk 1]
```diff
@@ -8,18 +8,14 @@ Feature: request-history-file-detail-view
...
-    Scenario Outline: User can view more details on a previously processed file
-        Given I process a file '<file>' through the icap server
-        And The transaction is available in the transaction log
-        When I click on the transaction record to open the detail view
-        Then the file detail view opens
-        And the file result details and the sanitisation issues content is displayed to show item '<issue>'
-        Examples:
-            | file        | issue    |
-            | issues.docx | 97976586 |
+    @TEST-169_2
```
Comment (asloman) [inline specs/features/request-history-file-detail-view.feature:11]: Two tests called TEST-169_2

File: specs/features/user-session-admin.feature [hunk 2]
```diff
@@ -3,16 +3,16 @@ Feature: user-session-admin
...
     Scenario Outline: User can log into the ui
         Given I am on the login screen
         When I enter login details '<username>' and '<password>'
         And I click login
         Then The home screen is displayed
         Examples:
-            | username             | password    |
-            | user@testuksouth.com | Password123 |
+            | username | password |
+            |          |          |
```
Comment (asloman) [inline specs/features/user-session-admin.feature:15]: Is this due to unsure what accounts are set up?

**Review by asloman — CHANGES_REQUESTED** [review]
Request History file detail feature has 2 tests called 169_2
Test login has blank username password? Are these meant to be or due to not having 1 consistent account set up yet?

File: specs/features/user-session-admin.feature (hunk 2 above)
Comment (NadHodeGW) [reply specs/features/user-session-admin.feature:15]: this is updated in next pr

**Review by asloman — APPROVED** [review]

**Pull request merged**